src/storage.rs

```diff
@@ -549,20 +551,25 @@ impl Cache {
                         file
                     })
                 })
-                .await?;
+                .await
+                .expect("failed to join");
```

Comment:
This join error is what happens if the spawned task panics, like when joining a thread.